client/getCountry.js

```diff
@@ -4,7 +4,7 @@ import countries from './countries.js';
 export function getFirstPathPart() {
   const url = new FtmUrl(window.location);
   const directories = url.pathname.split('/');
-  if (directories.length > 2) {
+  if (directories.length >= 2) {
     return directories[1];
   }
 
```

Comment:
can you log an error on line 26, so we can discern when this happens again through airbrake reporting?